### PR TITLE
feat: keycloak upgrade

### PR DESCRIPTION
**Pull request by j-zimnowoda**
## Checklist

- [ ] Architecture Design Records have been added as `adr/*.md` and appended to list in `adr/_index.md`, if applicable.
- [ ] The `values-schema.yaml` file and `test/**` fixtures have been updated to reflect code changes, if applicable.
- [ ] The OpenApi Schema from redkubes/otomi-api project is compatible with definitions from `values-schema.yaml` file, if applicable.
- [ ] Helm releases are meeting otomi's baseline security policies, if applicable.
- [ ] Helm chart and helmfile changes are tested against upgrade scenario, if applicable.

### CODE REVIEW

**Comment by j-zimnowoda** [comment]
There is some comment about this branch. See: #1283

**Comment by ElderMatt** [comment]
Where are the groups? I created 2 teams but they dont show up in keycloak, now I cannot assign users to a group.
image
image

**Review by ElderMatt — CHANGES_REQUESTED** [review]
At the moment Teams/groups are not automatically added in keycloak

**Comment by j-zimnowoda** [comment]
> At the moment Teams/groups are not automatically added in keycloak

Perhaps the Otomi-tasks needs to be update to use compatible client.

**Comment by Ani1357** [comment]
From the screenshot I see you are checking the master realm in keycloak instead of otomi. Please change the realm in the top left dropdown box and check again.

File: values-changes.yaml [hunk 1]
```diff
@@ -88,3 +88,8 @@ changes:
   - version: 14
     deletions:
       - 'apps.istio.meshConfig'
+  - version: 15
+    relocations:
+      - apps.keycloak.resources.postgresql: databases.keycloak.resources
+    deletions:
+      - apps.keycloak.postgresqlPassword
```
Comment (j-zimnowoda) [inline values-changes.yaml:95]: also
`- apps.resources. postgresql`

**Pull request merged**